pyaedt/misc/toolkit_manager.py

```diff
@@ -0,0 +1,283 @@
+
+
+def button_is_clicked(
+    install_action, toolkit_level, input_file, combo_toolkits, toolkit_name, install_button, uninstall_button
+):
+    """Install/Uninstall button action"""
+    # installation_option = option_action.get()
+    file = input_file.get()
+    selected_toolkit_name = combo_toolkits.get()
+    name = toolkit_name.get()
+
+    desktop = Desktop(
+        specified_version=version,
+        port=port,
+        new_desktop_session=False,
+        non_graphical=False,
+        close_on_exit=False,
+        student_version=student_version,
+    )
+
+    desktop.odesktop.CloseAllWindows()
+
+    if selected_toolkit_name != "Custom":
+        desktop.logger.info("TEST: VENV {}".format(venv_dir))
+        if is_toolkit_installed(selected_toolkit_name) and install_action:
+            desktop.logger.info("Updating {}".format(selected_toolkit_name))
+            desktop.add_custom_toolkit(selected_toolkit_name, file)
+            install_button.config(text="Update")
+            uninstall_button.config(state="normal")
+            desktop.logger.info("{} updated".format(selected_toolkit_name))
+        elif install_action:
+            desktop.logger.info("Installing {}".format(selected_toolkit_name))
+            desktop.add_custom_toolkit(selected_toolkit_name, file)
+            install_button.config(text="Update")
+            uninstall_button.config(state="normal")
+        elif is_toolkit_installed(selected_toolkit_name) and not install_action:
+            desktop.logger.info("Uninstalling {}".format(selected_toolkit_name))
+            desktop.add_custom_toolkit(selected_toolkit_name, install=False)
+            install_button.config(text="Install")
+            uninstall_button.config(state="disabled")
+            desktop.logger.info("{} uninstalled".format(selected_toolkit_name))
+        else:
+            desktop.logger.info("{} not installed".format(selected_toolkit_name))
+
+    else:
+        if install_action:
+            desktop.logger.info("Install {}".format(name))
+            if is_windows:
+                pyaedt_venv_dir = os.path.join(os.environ["APPDATA"], "pyaedt_env_ide", "v{}".format(version))
+                executable_interpreter = os.path.join(pyaedt_venv_dir, "Scripts", "python.exe")
+            else:
+                pyaedt_venv_dir = os.path.join(os.environ["HOME"], "pyaedt_env_ide", "v{}".format(version))
+                executable_interpreter = os.path.join(pyaedt_venv_dir, "bin", "python")
+            if os.path.isfile(executable_interpreter):
+                desktop.add_script_to_menu(
+                    toolkit_name=name,
+                    script_path=file,
+                    product=toolkit_level,
+                    executable_interpreter=executable_interpreter,
+                )
+            else:
+                desktop.logger.info("PyAEDT environment is not installed")
+        else:
+            desktop.logger.info("Uninstall {}".format(name))
```

Comment:
```suggestion
            desktop.logger.info("Uninstall {}.".format(name))
```
Not sure if this is where the period should be. All message strings should have concluding punctuation--which is virtually always a period.